pontoon/base/tests/test_views.py

```diff
@@ -334,7 +334,7 @@ def test_exclude_entities(self):
             'project': self.resource.project.slug,
             'locale': self.locale.code,
             'paths[]': [self.resource.path],
-            'excludeEntities[]': [self.entities[1].pk],
+            'exclude_entities': [self.entities[1].pk],
```

Comment:
This puzzles me. Why was it ``excludeEntities[]`` and not ``excludeEntities``? I assume it was a mistake, and the test actually doesn't really test this feature, sadly. :(

Reply:
@adngdb from what I recall that was because of the way Jquery serializes querystring parameters.

Reply:
That's the case for ``paths`` but ``exclude_entities`` is not a list, it's a string. The JS side does a ``.join(',')`` before sending it to the server, so really that ``[]`` shouldn't exist. I assume it was a leftover from a previous change.